js/app/src/components/File/File.svelte

```diff
@@ -1,3 +1,5 @@
+<svelte:options accessors={true} />
```

Comment:
Not sure if this has performance implications. I see that `gr.Textbox` has it so I figured it was ok.

This makes it super easy to test this change as I can just update the `value` myself. I'm also not sure how we could test this outside of an e2e test otherwise since the bug only arises when the value is updated by another component. Not the standard file upload.

Reply:
It has minor performance implications. I doubt it will cause any issues in reality.